docs/book/stacks-and-components/component-guide/model-deployers/model-deployers.md

```diff
@@ -87,46 +82,52 @@ zenml model-deployer register seldon --flavor=seldon \
    zenml stack register seldon_stack -m default -a aws -o default -d seldon
    ```
 
-2. Implements the continuous deployment logic necessary to deploy models in a way that updates an existing model server
-   that is already serving a previous version of the same model instead of creating a new model server for every new
-   model version. Every model server that the Model Deployer provisions externally to deploy a model is represented
-   internally as a `Service` object that may be accessed for visibility and control over a single model deployment. This
-   functionality can be consumed directly from ZenML pipeline steps, but it can also be used outside the pipeline to
-   deploy ad-hoc models. See the [seldon_model_deployer_step](https://sdkdocs.zenml.io/latest/integration_code_docs/integrations-seldon/#zenml.integrations.seldon.steps.seldon_deployer.seldon_model_deployer_step) for an example of using the Seldon Core Model Deployer to deploy a model inside a ZenML pipeline step.
-3. Acts as a registry for all Services that represent remote model servers. External model deployment servers can be
-   listed and filtered using a variety of criteria, such as the name of the model or the names of the pipeline and step
-   that was used to deploy the model. The Service objects returned by the Model Deployer can be used to interact with
-   the remote model server, e.g. to get the operational status of a model server, the prediction URI that it exposes, or
-   to stop or delete a model server:
+* Lifecycle Management: Provides mechanisms for comprehensive lifecycle management of model servers, including the ability to start, stop, and delete model servers, as well as to update existing servers with new model versions, thereby optimizing resource utilization and facilitating continuous delivery of model updates. Some core methods that can be used to interact with the remote model server include:
+
+`deploy_model` - Deploys a model to the serving environment and returns a Service object that represents the deployed model server.
+`find_model_server` - Finds and returns a list of Service objects that represent model servers that have been deployed to the serving environment, the 
```

Comment:
See below, but wondering about this reference to the DB... we don't talk about the database elsewhere... why does a user have to know?